### ISSUE TITLE
Shimmie2 hanging on image upload

### ISSUE THREAD

**Issue by IgnitionOverflow** [shish/shimmie2#520]
I've just had a nasty hard disk crash, so I'm setting up Shimmie2 fresh on a fresh server.

Shimmie2 installed fine, all tables in MySQL seem to be set up, folders created on the disk no problem, admin account created fine, I can even load extensions. However uploading an image causes the page to hang, I'm stuck on a blank white page perpetually.
I've tried Chrome and Firefox, I've even deleted and reinstalled Shimmie.
I've checked the disk and it appears to be writing something in the images folder, it created a folder and file for each image I try to upload just as it has before, but it won't add the MySQL entry with the tags, image IDs and such to the images table.
I've experimented with giving Shimmie root access to MySQL, I've tested using chmod 777 on the folders but the result is the same. After putting everything back I'm out of ideas.

I'm running an up to date Ubuntu Server 14.04.3LTS

Have I missed something?
Many Thanks!

![pmyadmin](https://cloud.githubusercontent.com/assets/14897394/10179981/93af086a-6701-11e5-9480-6b82c327bafd.png)

**Comment by JarJak** [comment]
check mysql and apache logs

**Comment by IgnitionOverflow** [comment]
thanks JarJak
I've checked the logs, in error.log there are a lot of warnings saying:

> [Warning] 'user' entry 'user@localhost' has both a password and an authentication plugin specified. The password will be ignored.

other than that I can't see anything that stands out...
I have plenty of space on disk as it's a fresh disk.
*How can I check if the created files are valid?*

I've tried uploading images today, and there are still no entries in the MySQL table, and there didn't appear to be anything in the logs, today's log file was blank... rebooting the server created some entries about the server shutting down and restarting in error.log, but the mysql.log logs are blank.

*other error messages in error.log:*
I've got "InnoDB memory heap is disabled", Shimmie doesn't need this does it?
"Plugin 'FEDERATED' is disabled." is this required?
there is a selection of other InnoDB warnings as well

I could try fiddling with InnoDB and see what happens..
any more help would be much appreciated
thanks

**Comment by jgen** [comment]
InnoDB isn't needed for Shimmie to work. (Source: I have a few site that run using a MySQL configuration with InnoDB disabled.)

You might also want to check your PHP configuration to make sure that file uploads are enabled and PHP can write to the temp location where the files are stored. (Also check the upload size limit)

**Comment by IgnitionOverflow** [comment]
thanks jgen

I've checked php.ini, file uploads are allowed and the limit is sufficient for the files I'm uploading, but I've bumped it up a bit all the same. I want to know if the pictures are uploading successfully, but I don't know how to check them, how can I convert it back to a file format I can view? I'm suspecting something interrupted Shimmie before the upload was completed as there are no entries in MySQL, but the account created successfully.
